libglusterfs/src/glusterfs/stack.h

```diff
@@ -65,6 +65,7 @@ struct _call_frame {
     call_frame_t *parent; /* previous BP */
     struct list_head frames;
     void *local;    /* local variables */
+    void *local_inode;  /*quotad lookup local inode*/
```

Comment:
This is not the right place to add a private field for an xlator. The `local` field is specifically created to store per-frame xlator's private data. You should use it.